src/benchmark/join_order_benchmark.cpp

```diff
@@ -113,5 +114,31 @@ int main(int argc, char* argv[]) {
   auto query_generator =
       std::make_unique<FileBasedQueryGenerator>(*benchmark_config, query_path, non_query_file_names, query_subset);
 
-  BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context}.run();
+  auto benchmark_runner =
+      BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context};
+
+  std::cout << "- Adding indexes to SQLite... " << std::flush;
+
+  if (benchmark_config->verify) {
+    // Add indexes to SQLite. This is a hack until we support CREATE INDEX ourselves and pass that on in the SQLiteWrapper
+    for (const auto& table_name : StorageManager::get().table_names()) {
+      // SQLite does not support adding primary keys, so we just add a regular index
+      std::cout << (std::string{"CREATE INDEX "} + table_name + "_primary ON " + table_name + "(id)") << std::endl;
```

Comment:
`+ --> <<`

Reply:
Ah. Copy and paste from the actual string for debugging. Do we even want it in here? I'd drop it.